validator/src/cli/thread_args.rs

```diff
@@ -0,0 +1,42 @@
+//! Arguments for controlling the number of threads allocated for various tasks
+
+use {
+    clap::Arg, solana_clap_utils::input_validators::is_within_range,
+    solana_rayon_threadlimit::get_max_thread_count,
```

Comment:
Keep using this for now, rip it out soon. If I make enough progress before we cut `v2.0` branch, can maybe delete this crate and make the ENV variable deprecated (given that someone would have an alternative way to tune everything)